test/followers_institution_test.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""Institution follower handler test."""
+from test_base import TestBase
+from models.user import User
+from models.institution import Institution
+from handlers.institution_followers_handler import InstitutionFollowersHandler
+
+
+class InstitutionFollowersHandlerTest(TestBase):
+    """Test the institution_followers_handler class."""
+
+    @classmethod
+    def setUp(cls):
+        """Provide the base for the tests."""
+        cls.test = cls.testbed.Testbed()
+        cls.test.activate()
+        cls.policy = cls.datastore.PseudoRandomHRConsistencyPolicy(
+            probability=1)
+        cls.test.init_datastore_v3_stub(consistency_policy=cls.policy)
+        cls.test.init_memcache_stub()
+        cls.ndb.get_context().set_cache_policy(False)
+        app = cls.webapp2.WSGIApplication(
+            [("/api/institution/(.*)/followers",
+                InstitutionFollowersHandler)
+             ], debug=True)
+        cls.testapp = cls.webtest.TestApp(app)
+        initModels(cls)
+
+    def test_post(self):
+        """Test the institution_follower_handler post method."""
+        # Pretend an authentication
+        self.os.environ['REMOTE_USER'] = 'mayzabeel@gmail.com'
+        self.os.environ['USER_EMAIL'] = 'mayzabeel@gmail.com'
+        # Verified objects, are empty
+        self.assertTrue(len(self.certbio.followers) == 0, "The number of followers expected was 0")
+        self.assertTrue(len(self.mayza.follows) == 0, "The number of follows expected was 0")
+        # Call the post method
+        self.testapp.post("/api/institution/%s/followers" % self.certbio.key.urlsafe())
+
+        # Update the objects
+        self.mayza = self.mayza.key.get()
+        self.certbio = self.certbio.key.get()
+
+        # An institution have 1 follower
+        self.assertTrue(len(self.mayza.follows) == 1, "The number of follows expected was 1")
+        # An user have 1 follow
+        self.assertTrue(len(self.certbio.followers) == 1, "The number of followers expected was 1")
+        # Institution have mayza in followers
+        self.assertTrue(self.mayza.key in self.certbio.followers, "Mayze should be in institution followers")
+        self.assertTrue(self.certbio.key in self.mayza.follows, "SpLab should be in user follows")
+
+        # Call the post method again
+        self.testapp.post("/api/institution/%s/followers" % self.certbio.key.urlsafe())
+        # Confirmed that follow only one time
+        self.assertTrue(len(self.certbio.followers) == 1, "The number of followers expected was 1")
```

Comment:
assertEquals, repete nos demais testes, sempre use assertEquals quando for usar o ==